### PR TITLE
change back icon

### PR DESCRIPTION
**Pull request by piensaenpixel**
@xavijam could you take a look?
back-icon
Fix - https://github.com/CartoDB/cartodb/issues/8065

### CODE REVIEW

File: app/assets/stylesheets/editor-3/menu.css.scss [hunk 1]
```diff
@@ -54,6 +54,38 @@
   width: 32px;
   height: 32px;
   margin: 0 auto 48px;
+
+  .arrow {
+    transform: translateX(30px) translateY(15px);
```
Comment (xavijam) [inline app/assets/stylesheets/editor-3/menu.css.scss]: could we use our transform mixin?

Comment (piensaenpixel) [inline app/assets/stylesheets/editor-3/menu.css.scss]: sure, let me check

**Comment by piensaenpixel** [comment]
pushed @xavijam

File: app/assets/stylesheets/editor-3/menu.css.scss [hunk 2]
```diff
@@ -56,35 +56,35 @@
...
   }
 
   .imago .arrowTop,
   .imago .arrowBottom {
-    transform-origin: left center;
-    transform: rotate(0deg);
-    transition: all 100ms ease-out 300ms;
+    @include transition(all 100ms ease-out 300ms);
+    @include transform(rotate(0deg));
+    @include css3-prefix(left center);
```
Comment (xavijam) [inline app/assets/stylesheets/editor-3/menu.css.scss]: I feel we miss something here

Comment (piensaenpixel) [inline app/assets/stylesheets/editor-3/menu.css.scss]: fuck

**Pull request merged**